specification/resources/resource-manager/Microsoft.Resources/preview/2019-10-01-preview/deploymentScripts.json

```diff
@@ -651,6 +656,16 @@
         }
       }
     },
+    "ContainerConfiguration": {
+      "type": "object",
+      "description": "Settings to customize ACI container instance.",
+      "properties": {
+        "containerGroupName": {
+          "type": "string",
+          "description":  "Container group name, if not specified then the name will get auto-generated."
```

Comment:
nit - Maybe add some information about when to provide a containerGroupName?

Reply:
+1 to that.

Reply:
I would also make sure that we have an example with and without `containerGroupName`.

---
In reply to: [395333253](https://github.com/Azure/azure-rest-api-specs/pull/8771#discussion_r395333253) [](ancestors = 395333253)

Reply:
Sounds good. I'll add an example with an without containerGroupName, good suggestion.
Re: name rules, these are basically the same from ACI I'll make sure to add the rules as part of the description.
Thanks!

Reply:
Additional info added. Thanks!